homeassistant/components/intellifire/translations/en.json

```diff
@@ -1,28 +1,37 @@
 {
-    "config": {
-        "abort": {
-            "already_configured": "Device is already configured",
-            "not_intellifire_device": "Not an IntelliFire Device."
-        },
-        "error": {
-            "cannot_connect": "Failed to connect"
-        },
-        "flow_title": "{serial} ({host})",
-        "step": {
-            "dhcp_confirm": {
-                "description": "Do you want to setup {host}\nSerial: {serial}?"
-            },
-            "manual_device_entry": {
-                "data": {
-                    "host": "Host"
-                },
-                "description": "Local Configuration"
-            },
-            "pick_device": {
-                "data": {
-                    "host": "Host"
-                }
-            }
-        }
-    }
+	"config": {
```

Comment:
Did you manually create this file ? You can run `python3 -m script.translations develop` to generate this.

Reply:
I think I did manually generate this ... I've committed an auto-generated one.